internal/configurator/builder_test.go

```diff
@@ -201,6 +201,43 @@ func TestShardingIndex(t *testing.T) { //nolint: paralleltest
 	}
 }
 
+func TestChartVersion(t *testing.T) { //nolint: paralleltest
+	t.Setenv(configurator.LicenseKeyEnvKey, "fake")
+
+	testCases := []struct {
+		name     string
+		expected string
+		setEnv   func()
+	}{
+		{
+			name:     "IsSetFromEnvVar",
+			expected: "1.0.1",
+			setEnv: func() {
+				t.Setenv(configurator.ChartVersionEnvKey, "1.0.1")
+			},
+		},
+		{
+			name:     "IsEmpty",
+			expected: "",
+			setEnv: func() {
+				t.Setenv(configurator.ChartVersionEnvKey, "")
+			},
+		},
+	}
```

Comment:
Nit: we may include a testcase showing what happens if the environment variable is not set but the corresponding key in the configuration was set (this could be done by setting it up in the yaml config)

Reply:
I set it as `ChartVersion string yaml:"-"` as I thought this value as something done automatically and not configurable from the yaml. WDYT?